db/migrate/20150609193512_create_songs.rb

```diff
@@ -0,0 +1,14 @@
+class CreateSongs < ActiveRecord::Migration
+  def change
+  	enable_extension 'uuid-ossp'
+
+    create_table :songs, id: :uuid do |t|
+      t.string :name
+      t.string :artist
+      t.string :spotify_uri
+      t.timestamps null: false
```

Comment:
We usually move the timestamps up to the top, to keep our schema.rb file relatively organized.